### PR TITLE
Code cleaning

### PR DESCRIPTION
**Pull request by RUBADAYYEH**

### CODE REVIEW

File: AirportTicketBookingSystem/Domain/FlightManagement/Flight.cs [hunk 1]
```diff
@@ -5,20 +5,23 @@
...
-    public  class Flight
+    public record Flight
     {
-       [Required(ErrorMessage ="flight id must be entered.")]
-        
-       public  int FlightId { get; init; }
+        [Required(ErrorMessage = "flight id must be entered.")]
+
+        public int FlightId { get; init; }
         public decimal Price { get; set; }
```
Comment (montss) [inline AirportTicketBookingSystem/Domain/FlightManagement/Flight.cs:13]: why set not init ?

File: AirportTicketBookingSystem/Domain/UserManagement/Filter.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AirportTicketBookingSystem.Domain.UserManagement
+{
+    internal class Filter:Utilities
```
Comment (montss) [inline AirportTicketBookingSystem/Domain/UserManagement/Filter.cs]: format the document

File: AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+namespace AirportTicketBookingSystem.Domain.UserManagement
+{
+    public class FilterHandler
+    {
+        public static List<Flight> HandleFilteringOnPrice(decimal price, List<Flight> filteredList)
+        {
+            Console.ForegroundColor = ConsoleColor.Cyan;
+
+            Console.WriteLine($"you have selected a maximum price {price}!");
```
Comment (montss) [inline AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs]: Upper case in the start

File: AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            Console.WriteLine($"you have selected a Flight Class {flightClass}!");
+            filteredList = Filter.FilterBasedOnClass(filteredList, flightClass);
+            return filteredList;
+        }
+
+        public static List<Flight> HandleFilteringOnDestinationAirport(string destinationAirport, List<Flight> filteredList)
+        {
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            if (destinationAirport != "")
```
Comment (montss) [inline AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs]: we should always use `{ }`

File: AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+                Console.WriteLine($"you have selected a destination Airport {destinationAirport}!");
+            filteredList = Filter.FilterBasedOnDepartureAirport(filteredList, destinationAirport);
+
+            return filteredList;
+        }
+
+        public static List<Flight> HandleFilteringOnDepartureCountry(string departureLocation, List<Flight> filteredList)
+        {
+            Console.ForegroundColor = ConsoleColor.Cyan;
+            if (departureLocation != "")
```
Comment (montss) [inline AirportTicketBookingSystem/Domain/UserManagement/FilterHandler.cs]: `departureLocation.length > 0, string.IsEmptyNullOrEmpty(departureLocation) , departureLocation.IsEmpty()`

File: AirportTicketBookingSystem/Utilities.cs [hunk 6]
```diff
@@ -14,53 +15,57 @@ namespace AirportTicketBookingSystem
...
+            StandardMessage.EscapeEventByAnyKeyMessage();
         }
         public static void LogIn()
         {
             Console.WriteLine("Write your username to enter: ");
             string? username = Console.ReadLine() ?? "";
-            if (username != null)
-            {
-                if (users.Where(u => u.UserName == username).Any()) // count vs any 
+            bool usernameIsNotNull=UserValidator.ValidateUserByUsername(username);
```
Comment (montss) [inline AirportTicketBookingSystem/Utilities.cs]: var instead of bool

**Review by montss — CHANGES_REQUESTED** [review]

File: AirportTicketBookingSystem/Domain/FlightManagement/Flight.cs (hunk 1 above)
Comment (RUBADAYYEH) [reply AirportTicketBookingSystem/Domain/FlightManagement/Flight.cs:13]: the price of the unbooked tickets might change in future as the time of the departure gets close.

**Pull request merged**